usaspending_api/etl/tests/integration/test_load_transactions_in_delta.py

```diff
@@ -240,11 +243,6 @@ def verify_transaction_ids(spark, expected_transaction_id_lookup, expected_last_
         with connection.cursor() as cursor:
             cursor.execute(f"SELECT setval('transaction_id_seq', {max_transaction_id}, false)")
 
-        if isinstance(expected_last_load, datetime):
```

Comment:
These assert statements were not checking anything updated or inserted by the `load_transactions_in_delta` Django command. For that reason, we are removing them because it enables us to, in some places, remove Postgres data initialization all together.